### PR TITLE
French translation for changelog

### PR DESCRIPTION
**Pull request by TheDauntless**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #< insert number here >.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit cd225e9c9243519dd840de7fe534a872fe7df57f: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

File: Document-fr/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+- Mise à jour des traductions française et espagnole.
+- Ajout d'une traduction chinoise et japonaise du journal des modifications.
+- Vérification automatisée de la syntaxe markdown et de l'accessibilité des URLs.
+- Ajout de codes d'identification aux exigences, qui seront inclus dans la future version du MSTG afin de trouver plus facilement les recommandations et les cas de test.
+- Réduction de la taille du référentiel et ajout de Generated au .gitignore.
+- Ajout d'un code de conduite et de directives contributives.
+- Ajout d'un modèle de PR.
+- Mise à jour de la synchronisation avec le référentiel utilisé pour l'hébergement du site Web sur Gitbook.
+- Mise à jour des scripts pour générer les fichiers XML / JSON / CSV pour toutes les traductions.
+
```
Comment (commjoen) [inline Document-fr/CHANGELOG.md]: I still miss the last one :D (something something foreword in ZHTW 👍 )

**Comment by OMTGreleaser** [comment]
Results for commit 3af04ab56acf038c6d6107c79bb240fa9e66f593: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit 53aa5770bc8181c442251cc3c150fe5bfa54e83a: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Pull request merged**